src/extensions/default/WebPlatformDocs/WebPlatformDocs.less

```diff
@@ -131,7 +131,7 @@
 }
 
 .divider-holder {
-    // This nesting lets us effectively have 'left: (37% + 12px)'
+    // This nesting lets us effectively have 'left: (37% + 12px)
```

Comment:
We removed the WebPlatformDocs extension, so you can get rid of the changes to this file altogether.